build.sbt

```
@@ -13,7 +13,7 @@ addCommandAlias("headerCheckAll", "; all root/headerCheck Test/headerCheck; inte
 val flywayVersion               = "11.3.0"
 val hikariVersion               = "6.2.1"
 val quillVersion                = "4.8.6"
-val sipiVersion                 = "v31.4.0"
+val knoraSipiBaseImage          = "v31.5.0"
```

```suggestion
val knoraSipiBaseImageVersion          = "v31.5.0"
```
Please also keep it sorted :)